schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    name varchar(100),
+    id int,
```

Comment:
- [OPTIONAL] You can make the id column not null
  <img width="291" alt="Screenshot 2023-06-13 at 11 52 35" src="https://github.com/Mylo16/db-query/assets/81444338/5ecc8913-a3e2-4bfc-a66f-1700d5c4e4ca">